### ISSUE TITLE
Everything getting marked unknown after disambiguation

### ISSUE THREAD

**Issue by IvyJL** [Hedera-Lang-Learn/hedera#460]
A few unknown words were displayed. Then I disambiguated one. And suddenly everything went unknown.
<img width="1148" alt="Everything unknown after disambiguation" src="https://user-images.githubusercontent.com/18600875/201169237-8f18db6b-a9a5-4c0d-8146-e03aac0c87d4.png">

**Comment by IvyJL** [comment]
So a list is selected and "show unknown" is clicked. But when you disambiguate something, it's like the list gets unselected, the percents go back to 0% and everything is considered unknown. If you reselect the list, then you get updated percents. I thought it used to just update the percentage of the selected list when you disambiguated.

**Comment by bilbe** [comment]
Yes, I noticed yesterday that selecting my personal list showed everything as 0% and red, when, if fact, I new all the words. Definitely needs fixing

**Comment by bilbe** [comment]
This is still broken. The way to reproduce this is to select a vocab list, select a word, and then disambiguate the word. At that moment all the words turn red.

**Comment by IvyJL** [comment]
Seems to be fixed! Thanks!
